app/components/blacklight/document_metadata_component.rb

```diff
@@ -9,8 +9,10 @@ class DocumentMetadataComponent < Blacklight::Component
 
     # @param fields [Enumerable<Blacklight::FieldPresenter>] Document field presenters
     # rubocop:disable Metrics/ParameterLists
-    def initialize(fields: [], tag: 'dl', classes: %w[document-metadata dl-invert row], show: false, view_type: nil, field_layout: nil, **component_args)
+    def initialize(fields: [], document: nil, presenter: nil, tag: 'dl', classes: %w[document-metadata dl-invert row], show: false, view_type: nil, field_layout: nil, **component_args)
       @fields = fields
+      @document = document
```

Comment:
What are these ivars used for? Is the intention to make them available to people who subclass this?  If so, can you leave a comment about that so that they aren't refactored out by accident?